### PR TITLE
homepage added

### PR DESCRIPTION
**Pull request by shrey27**
Homepage screen for E-commerce project.
Features:
- Responsive navbar
- Mobile view -> drawer for categories
- submenu for showing cart, wishlist and account pages links
- carousal(takes few seconds to load)
- Top deals (using card component)

### CODE REVIEW

File: Static-Pages/homepage/index.html [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+            class="header__image"
+            src="https://images.unsplash.com/photo-1621057621391-7ed446a24b41?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTR8fHByb3RlaW58ZW58MHwwfDB8fA%3D%3D&auto=format&fit=crop&w=600&q=60"
+            alt="adv_1"
+          />
+          <img
+            class="header__image"
+            src="https://images.unsplash.com/photo-1591311630200-ffa9120a540f?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mnx8Z3ltJTIwZXF1aXBtZW50fGVufDB8MHwwfHw%3D&auto=format&fit=crop&w=600&q=60"
+            alt="adv_2"
+          />
+        </div> -->
```
Comment (lomesshh) [inline Static-Pages/homepage/index.html]: remove the commented code

File: Static-Pages/homepage/homepage.css [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+}
+.menu__btn__name {
+  display: none;
+}
+.submenu {
+  display: none;
+  position: absolute;
+  top: 2rem;
+  right: 0rem;
+  z-index: 110;
```
Comment (lomesshh) [inline Static-Pages/homepage/homepage.css]: reduce the value of z-index.
110 is too much

**Review by lomesshh — APPROVED** [review]
Just do those 2 small changes
rest all looks good to me

File: Static-Pages/homepage/index.html (hunk 1 above)
Comment (shrey27) [reply Static-Pages/homepage/index.html]: sure

**Review by rohit8686 — APPROVED** [review]
Looks good !

**Comment by SJTGSHIVAM** [comment]
@shrey27 everything looks good just one issue I see is with the sizing of cards which I even tried to fix as discussed over the call.
you can employ flowing piece of CSS to fix it.
deal--cards {
  display: grid;
  grid-template-columns: repeat(auto-fit, minmax(300px, auto));
  justify-content: center;
  gap: 1rem;
}
its just playfully value which I tried you can put in your desired values there.

Even this idon't find a breaking change so doing it or not is completely up to you.
I believe this code looks good and we can surely merge it.
Good to go !

**Review by SJTGSHIVAM — APPROVED** [review]
Looks Great!

**Pull request merged**